packages/core/src/lib/__tests__/modal.spec.ts

```diff
@@ -24,15 +24,41 @@ describe('Modal', () => {
     const onClose = vi.fn();
     component.$on('close', onClose);
 
-    const background = container.querySelector('div[role="button"]');
-    if (!background) {
-      throw new Error('Background not found');
+    const modal = container.querySelector('.relative.max-w-lg');
+    if (!modal) {
+      return;
     }
```

Comment:
If your tests couldn't find an element for the modal, the test should fail. In this case, it'd be fine to move right on the the `click`. This is a side-effect of using `querySelector` in a test instead of `screen.getByRole` or similar, which _will_ error if the element can't be found

Things you should almost never do in unit tests, in descending order of "don't do this":

- Early returns
- Throwing your own errors
- `if` statements
- `for` loops